### PR TITLE
Add kraken extension

### LINKED ISSUE #177: Kraken support
Hi,

Could is it possible to re-add the support of kraken in v.4 ?

Thanks!

### PR DESCRIPTION
**Pull request by Alexandre-io**
- This PR add the kraken extension for zenbot 4

Special thanks to @crubb, @nibzo1

close #177

### CODE REVIEW

File: extensions/kraken/update-products.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+var KrakenClient = require('kraken-api');
+var kraken = new KrakenClient();
+
+var mapping;
+var products = [];
+
+function addProduct(base, quote, altname) {
+    products.push({
+        asset: base,
```
Comment (DeviaVir) [inline extensions/kraken/update-products.sh:11]: not sure if necessary, but I noticed it in the GDAX `products`:
https://github.com/carlos8f/zenbot/pull/205/files#diff-4899b562666048f7641da71b0434f132R15

might need to add `id`?

**Review by DeviaVir — APPROVED** [review]
great work!

File: extensions/kraken/update-products.sh (hunk 1 above)
Comment (DeviaVir) [reply extensions/kraken/update-products.sh:11]: actually, upon closer inspection I don't think this is necessary (anymore).

**Comment by Alexandre-io** [comment]
New PR #207 without squash to allow contributors to be properly attributed.

**Pull request closed without merging**